docs/extensions/jvm.md

```diff
@@ -0,0 +1,50 @@
+---
+id: jvm
+title: JVM Extension
+sidebar_label: JVM
+---
+
+The following section contains JVM-specific extensions to the build server
+protocol.
+
+
+## Test Environment Request
```

Comment:
Any opinion if we should also add generic `buildTarget/jvmEnvironment` here as well?
@jvican @olafurpg